src/core/scene/a-scene.js

```diff
@@ -389,9 +412,9 @@ module.exports = registerElement('a-scene', {
                 window.performance.mark('render-started');
               }
               sceneEl.clock = new THREE.Clock();
-              sceneEl.render();
               sceneEl.renderStarted = true;
               sceneEl.emit('renderstart');
+              sceneEl.render();
```

Comment:
Ok I had this to notify the post proc system that the rendertargets are ready in case it wants to set a depth texture. That was when aframe was using an older version of three, now we can just attach the depth texture on the render target when creating them